rllib/algorithms/algorithm.py

```diff
@@ -936,10 +939,20 @@ def duration_fn(num_units_done):
             if self.reward_estimators:
                 # Compute off-policy estimates
                 metrics["off_policy_estimator"] = {}
-                total_batch = concat_samples(all_batches)
```

Comment:
We shouldn't concatenate and then split by episode here. In case of bandits we don't even need to split_by_episodes since each rows is already one episode. In case of RL, each batch is already ended at some episode so we gain nothing by concating all batches together.